packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -363,6 +366,55 @@ This behavior is slightly different in the highlight selection style, where sing
 </Flex>
 ```
 
+
+### Action bar <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+An ActionBar can be used when a user needs to perform actions on one or more items at the same time.
```

Comment:
Should ActionBar be a link to its docs?

Reply:
We should definitely add that after the ActionBar docs get merged and are public. These might go out before that though.

Reply:
Added back since we decided we'll merge this after the ActionBar docs are public